src/types/Props.ts

```diff
@@ -0,0 +1,7 @@
+import { Tab } from './Tab';
+
+export interface Props {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void,
```

Comment:
Add comas or dots (follow your style-guide):
```suggestion
  tabs: Tab[],
  selectedTabId: string,
  onTabSelected: (tab: Tab) => void,
```